### ISSUE TITLE
NVDA does not read the name of folders after navigating several folders

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#5857]
After navigating several folders NVDA no longer reads the names of folders open.
To reproduce this problem:
1) Open the Start menu, press the arrow down to the item "this computer", press enter.
2) Choose a computer unit, in my case I chose drive d :.
3) Choose a folder within this unit, preferably a folder where there are several subfolders and press enter.
4) Now navigate several times between folders and subfolders with arrows, enter and back space.
5) After some navigations in my case I sailed with enter and back space 10 times, pressing incert + t NVDA reads only explorer, not read the folder names.
A log file set for input and output are attached.
I apologize for the language to be in Portuguese, but when looking at the log line occurrences
Input: kb (laptop): NVDA + t
You will direct for all the times that NVDA read the folder name correctly, and sometimes he just read:
Speaking [u'explorer ']
A mistake aparesse in the log that caught my attention was also:
oleacc.AccessibleObjectFromEvent with window 1312354, objectID 2817 and childID 0: [Error -2147220991] An event 
[nvda.log.txt](https://github.com/nvaccess/nvda/files/203757/nvda.log.txt)
could not call subscribers

Again thank you for your attention.

**Comment by jcsteh** [comment]
Somehow, when your system gets into this state, some Google Chrome window has the focus. Do you perhaps have the preview pane enabled? You can check in the View tab of the ribbon. Certain previews seem to cause problems for screen readers.

**Comment by fernando-jose-silva** [comment]
good afternoon everyone.
Thanks for attention.
I made tests with closed chrome and unchecking all panels and Explorer toolbars and the same symptom continues.

**Comment by derekriemer** [comment]
Hi,
When this happens, do you mind pasting the dev info?

On 4/6/2016 10:49 AM, fernando-jose-silva wrote:
>
> good afternoon everyone.
> Thanks for attention.
> I made tests with closed chrome and unchecking all panels and Explorer 
> toolbars and the same symptom continues.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/nvaccess/nvda/issues/5857#issuecomment-206460941>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194

**Comment by fernando-jose-silva** [comment]
Good afternoon, I do not know if correctly understood your request, which is a developer of information?
The following is a log attached to the time the error ever happens.
Thank you very much.
[nvda-explorer.txt](https://github.com/nvaccess/nvda/files/207014/nvda-explorer.txt)

**Comment by jcsteh** [comment]
When this occurs, please press NVDA+f1. This will open the Log Viewer. Then, copy the information from the cursor to the end of the document and paste it here. This is what is meant by dev info.

@derekriemer: For future reference, most users won't know what dev info means. We might need to provide a reference for this, but for now, you'll need to provide details. Thanks.

**Comment by fernando-jose-silva** [comment]
Good evening, I believe that this is expected, only the log snippet that shows starting from the moment I pressed enter on the d: \ drive the freezing of NVDA and press NVDA + t pronunciation of the name of the window as explorer .
IO - inputCore.InputManager.executeGesture (20:36:02):
Input: kb(laptop):enter
DEBUGWARNING - watchdog._watcher (20:36:02):
Trying to recover from freeze, core stack:
  File "nvda.pyw", line 191, in <module>
  File "core.pyc", line 382, in main
  File "wx\_core.pyc", line 8657, in MainLoop
  File "wx\_core.pyc", line 7952, in MainLoop
  File "core.pyc", line 352, in Notify
  File "IAccessibleHandler.pyc", line 884, in pumpAll
  File "IAccessibleHandler.pyc", line 617, in processGenericWinEvent
  File "IAccessibleHandler.pyc", line 521, in winEventToNVDAEvent
  File "_UIAHandler.pyc", line 279, in isUIAWindow
  File "_UIAHandler.pyc", line 273, in _isUIAWindowHelper

DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:36:02):
oleacc.AccessibleObjectFromEvent with window 918812, objectID 327 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:36:02):
oleacc.AccessibleObjectFromEvent with window 918812, objectID 328 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:36:02):
oleacc.AccessibleObjectFromEvent with window 918812, objectID 329 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - watchdog._watcher (20:36:03):
Trying to recover from freeze, core stack:
  File "nvda.pyw", line 191, in <module>
  File "core.pyc", line 382, in main
  File "wx\_core.pyc", line 8657, in MainLoop
  File "wx\_core.pyc", line 7952, in MainLoop
  File "core.pyc", line 352, in Notify
  File "IAccessibleHandler.pyc", line 872, in pumpAll
  File "IAccessibleHandler.pyc", line 668, in processFocusWinEvent
  File "IAccessibleHandler.pyc", line 684, in processFocusNVDAEvent
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 110, in _getPropertyViaCache
  File "NVDAObjects\IAccessible\__init__.pyc", line 633, in _get_shouldAllowIAccessibleFocusEvent
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 110, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 592, in _get_parent

DEBUGWARNING - core.Notify (20:36:03):
errors in this core pump cycle
Traceback (most recent call last):
  File "core.pyc", line 352, in Notify
  File "IAccessibleHandler.pyc", line 872, in pumpAll
  File "IAccessibleHandler.pyc", line 668, in processFocusWinEvent
  File "IAccessibleHandler.pyc", line 684, in processFocusNVDAEvent
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 110, in _getPropertyViaCache
  File "NVDAObjects\IAccessible\__init__.pyc", line 633, in _get_shouldAllowIAccessibleFocusEvent
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 110, in _getPropertyViaCache
  File "watchdog.pyc", line 198, in _COMError_init
CallCancelled
IO - speech.speak (20:36:03):
Speaking [u'Modo de Exibi\xe7\xe3o de Itens  lista']
IO - inputCore.InputManager.executeGesture (20:36:10):
Input: kb(laptop):NVDA+t
IO - speech.speak (20:36:10):
Speaking [u'explorer']

**Comment by jcsteh** [comment]
Please press NVDA+f1 once this occurs. NVDA+f1 causes additional information to be logged about the current object. This is the info we are looking for.

**Comment by fernando-jose-silva** [comment]
that
thank you so much
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (21:51:14):
Developer info for navigator object:
name: u'\xc1rea de trabalho'
role: ROLE_WINDOW
states: 
isFocusable: False
hasFocus: False
Python object: <NVDAObjects.window.Desktop object at 0x05126350>
Python class mro: (<class 'NVDAObjects.window.Desktop'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: ''
location: (0, 0, 1366, 768)
value: ''
appModule: <'appModuleHandler' (appName u'csrss', process ID 712) at address 5126390>
appModule.productName: exception: processHandle is 0
appModule.productVersion: exception: processHandle is 0
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 65552
windowClassName: u'#32769'
windowControlID: 0
windowStyle: -1778384896
windowThreadID: 820
windowText: u''
displayText: u''

**Comment by ehollig** [comment]
@fernando-jose-silva, is this still an issue?

**Comment by fernando-jose-silva** [comment]
no thank you very much.

**Comment by derekriemer** [comment]
closing as fixed.
Thanks.
